webportal/lambda/.chalice/config.json

```diff
@@ -3,42 +3,83 @@
+        "TABLENAME_MANAGE_DISK_INDEX": "dev-diskspace-username-index",
+        "TABLENAME_MANAGE_UPTIME": "dev-ScheduleUptimeTable",
+        "TABLENAME_MANAGE_UPTIME_INDEX":"dev-scheduleuptime-username-index",
+        "COGNITO_USER_POOL":"dev-sdc-dot-cognito-pool",
+        "IDP_PROVIDER_ARNS":"arn:aws:cognito-idp:us-east-1:911061262852:userpool/us-east-1_Y5JI7ysvY"
+      }
+    },
+    "test": {
+      "api_gateway_stage": "api",
+      "environment_variables":{
+        "TABLENAME_USER_STACKS": "dev-UserStacksTable",
+        "TABLENAME_AVAILABLE_DATASET": "dev-AvailableDataset",
+        "RESTAPIID": "z2jk3bjo2m",
+        "AUTHORIZERID": "zv7cbg",
+        "TABLENAME_TRUSTED_USERS": "dev-TrustedUsersTable",
+        "TABLENAME_EXPORT_FILE_REQUEST": "dev-RequestExportTable",
+        "TABLENAME_MANAGE_USER": "dev-ManageUserWorkstationTable",
+        "TABLENAME_MANAGE_USER_INDEX": "dev-workstation-username-index",
+        "TABLENAME_MANAGE_DISK": "dev-ManageDiskspaceRequestsTable",
+        "TABLENAME_MANAGE_DISK_INDEX": "dev-diskspace-username-index",
+        "TABLENAME_MANAGE_UPTIME": "dev-ScheduleUptimeTable",
+        "TABLENAME_MANAGE_UPTIME_INDEX":"dev-scheduleuptime-username-index",
+        "COGNITO_USER_POOL":"test-sdc-dot-cognito-pool",
+        "IDP_PROVIDER_ARNS":"arn:aws:cognito-idp:us-east-1:911061262852:userpool/us-east-1_89P1HiXoV"
       }
     },
     "prod": {
       "api_gateway_stage": "api",
       "environment_variables":{
-        "TABLENAME_USER_STACKS": "",
-        "TABLENAME_AVAILABLE_DATASET": "",
-        "RESTAPIID": "",
-        "AUTHORIZERID": "",
-        "TABLENAME_TRUSTED_USERS": "",
-        "TABLENAME_EXPORT_FILE_REQUEST": "",
-        "TABLENAME_MANAGE_USER": "",
-        "TABLENAME_MANAGE_USER_INDEX": "",
-        "TABLENAME_MANAGE_DISK": "",
-        "TABLENAME_MANAGE_DISK_INDEX": "",
-        "TABLENAME_MANAGE_UPTIME": "",
-        "TABLENAME_MANAGE_UPTIME_INDEX":""
+        "TABLENAME_USER_STACKS": "prod-UserStacksTable",
+        "TABLENAME_AVAILABLE_DATASET": "prod-AvailableDataset",
+        "RESTAPIID": "cyfca70occ",
+        "AUTHORIZERID": "r6ck6c",
+        "TABLENAME_TRUSTED_USERS": "production-TrustedUsersTable",
+        "TABLENAME_EXPORT_FILE_REQUEST": "production-RequestExportTable",
+        "TABLENAME_MANAGE_USER": "production-ManageUserWorkstationTable",
+        "TABLENAME_MANAGE_USER_INDEX": "production-workstation-username-index",
+        "TABLENAME_MANAGE_DISK": "production-ManageDiskspaceRequestsTable",
+        "TABLENAME_MANAGE_DISK_INDEX": "production-diskspace-username-index",
+        "TABLENAME_MANAGE_UPTIME": "production-ScheduleUptimeTable",
+        "TABLENAME_MANAGE_UPTIME_INDEX":"production-scheduleuptime-username-index",
+        "COGNITO_USER_POOL":"dev-sdc-dot-cognito-pool",
+        "IDP_PROVIDER_ARNS":"arn:aws:cognito-idp:us-east-1:911061262852:userpool/us-east-1_Y5JI7ysvY"
+      }
+    },
+    "production": {
```

Comment:
2 things to note here... First off, I'm committing these configs to git, because I accidentally blew away a recently added config in development. I would rather not have a manual step each time that is fraught with the potential to make prod non-functional.

Secondly, there is a reason there is "prod" and "production". "prod" currently points to the dev Cognito user pool. "production" will actually point to the production Cognito user pool. Once "production" is deployed, the angular app can be pointed away from "prod" and to "production". This will allow a seamless invisible transition to the correct user pool.